src/include/platform/CHIPDeviceEvent.h

```diff
@@ -171,7 +171,7 @@ enum PublicEventTypes
      * When supportsConcurrentConnection = False, the ConnectNetwork command cannot start until
      * the BLE device is closed and the WiFi device has been started.
      */
-    kWiFiDeviceAvailable,
+    kOperationalNetworkStarted,
```

Comment:
This is a change of public API. Changing this may break platforms. This was the wrong name, but the old name must be maintained, and kWiFiDeviceAvailable should just steer to the same place as kOperationalNetworkStarted.

Reply:
Done, however as this was only added a few weeks ago after the 1.2 release, I do not think it has been used by anyone, so we may not need to do it.